app/models/ticket_level.rb

```diff
@@ -48,6 +48,10 @@ def event_discounts
     self.amount = 0 if amount.nil?
   end
 
+  def amount_as_money
+    Amount.new(amount||0, nonprofit.currency)
+  end
```

Comment:
I think it's a good idea to organize the money like this. In the future we could move this to a concern.

Reply:
Agreed! Eventually, I think we should use the Money gem which will handle some of this stuff.